cmake/kokkos_compiler_id.cmake

```diff
@@ -155,32 +155,16 @@ if(KOKKOS_CXX_COMPILER_ID STREQUAL Fujitsu)
 endif()
 
 # Enforce the minimum compilers supported by Kokkos.
-if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 17)
-endif()
-if(CMAKE_CXX_STANDARD EQUAL 17)
-  set(KOKKOS_CLANG_CPU_MINIMUM 8.0.0)
-  set(KOKKOS_CLANG_CUDA_MINIMUM 10.0.0)
-  set(KOKKOS_CLANG_OPENMPTARGET_MINIMUM 15.0.0)
-  set(KOKKOS_GCC_MINIMUM 8.2.0)
-  set(KOKKOS_INTEL_LLVM_CPU_MINIMUM 2021.1.1)
-  set(KOKKOS_INTEL_LLVM_SYCL_MINIMUM 2024.2.1)
-  set(KOKKOS_NVCC_MINIMUM 11.0.0)
-  set(KOKKOS_HIPCC_MINIMUM 5.2.0)
-  set(KOKKOS_NVHPC_MINIMUM 22.3)
-  set(KOKKOS_MSVC_MINIMUM 19.29)
-else()
-  set(KOKKOS_CLANG_CPU_MINIMUM 14.0.0)
-  set(KOKKOS_CLANG_CUDA_MINIMUM 14.0.0)
-  set(KOKKOS_CLANG_OPENMPTARGET_MINIMUM 15.0.0)
-  set(KOKKOS_GCC_MINIMUM 10.4.0)
-  set(KOKKOS_INTEL_LLVM_CPU_MINIMUM 2022.0.0)
-  set(KOKKOS_INTEL_LLVM_SYCL_MINIMUM 2024.2.1)
-  set(KOKKOS_NVCC_MINIMUM 12.0.0)
-  set(KOKKOS_HIPCC_MINIMUM 5.2.0)
-  set(KOKKOS_NVHPC_MINIMUM 22.3)
-  set(KOKKOS_MSVC_MINIMUM 19.30)
-endif()
+set(KOKKOS_CLANG_CPU_MINIMUM 14.0.0)
+set(KOKKOS_CLANG_CUDA_MINIMUM 14.0.0)
```

Comment:
```suggestion
set(KOKKOS_CLANG_CUDA_MINIMUM 16.0.0)
```

Reply:
Please undo or change it to 15.
We currently have a CI build with Clang15